ros2cli/ros2cli/node/daemon.py

```diff
@@ -132,5 +132,24 @@ def __exit__(self, exc_type, exc_value, traceback):
         self._proxy.__exit__(exc_type, exc_value, traceback)
 
 
+def shutdown_daemon(args, wait_until_shutdown=None):
```

Comment:
nit: `wait_until_shutdown` doesn't sound clear enough to me, I'm not sure if it's a duration or a time point.
Maybe "wait_duration" or "wait_for_shutdown_duration" is more clear.
A docstring would also be helpful.

Reply:
Sounds good. See https://github.com/ros2/ros2cli/pull/620/commits/1633a924d5c6da1586f4ad4c7d9015d1361c1b39.